owners/OWNERS.example.yaml

```diff
@@ -18,7 +23,8 @@
 
 # GitHub teams are also supported as owners. When a team is declared as an
 # owner, it is treated the same as if all members of the team were listed as
-# owners individually
+# owners individually (see AMPHTML teams
+# [here](https://github.com/orgs/ampproject/teams)).
```

Comment:
Hyperlinks don't show up in YAML file. Plain text link should suffice.

Reply:
Done.